api/src/services/__tests__/competenciesService.ts

```diff
@@ -62,4 +64,28 @@ describe('competenciesService', () => {
       });
     });
   });
+
+  describe('updateCompetency', () => {
+    it('should update the specified competency if the current user is the one who originally authored the competency', async () => {
```

Comment:
This test description says "if the current user is the one who originally authored the competency" but there is no code in the test that asserts anything about the "current user". Looks like the test description needs to be updated.

Reply:
Updated test description to `'should update the specified/selected competency'`